frontend/app/error.vue

```diff
@@ -13,14 +13,20 @@ onMounted(() => {
 })
 
 const errorMessage = computed(() => {
-  switch (props.error?.statusCode) {
-    case 404:
+  const error = props.error
+  if (!error) {
+    return 'Nieznany błąd'
+  }
+  const message = error.message?.toString().toLowerCase() || ''
+  if (error.statusCode === 404) {
+    if (message.includes('page not found')) {
       return 'Uwaga rycerzu,\n ta strona zniknęła jak zamek w chmurach.\n Wróć na właściwą drogę!'
-    case 500:
-      return 'Rycerz napotkał przeszkodę\n na swojej drodze.\n Spróbuj ponownie za chwilę.'
-    default:
-      return props.error?.message
+    }
+  }
+  if (error.statusCode === 500) {
+    return 'Rycerz napotkał przeszkodę\n Na swojej drodze.\n Spróbuj ponownie później.'
   }
+  return error.statusMessage
```

Comment:
You probably want to return `.message`, here because now I get
<img width="1270" height="598" alt="image" src="https://github.com/user-attachments/assets/c2f18e48-9ea2-44d4-8fb1-f6cf0297eb75" />